src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -315,4 +315,9 @@ pub const CONFIG_CHANGE_HISTORY: &[ChangeInfo] = &[
         severity: ChangeSeverity::Info,
         summary: "`build.vendor` is now enabled by default for dist/tarball sources when 'vendor' directory and '.cargo/config.toml' file are present.",
     },
+    ChangeInfo {
+        change_id: 134650,
+        severity: ChangeSeverity::Warning,
+        summary: "Removed `rust.parallel-compiler` as it was deprecated long time ago.",
```

Comment:
This reads a bit weirdly to me; what about 'was deprecated in #xxxxx' (or a PR link)?

Reply:
Makes sense